qa/rpc-tests/getchaintips.py

```diff
@@ -27,6 +27,7 @@ def run_test (self):
         # Split the network and build two chains of different lengths.
         self.split_network ()
         self.nodes[0].generate(10)
+        self.nodes[2].sendtoaddress(self.nodes[2].getnewaddress(), 1)
```

Comment:
this splits the two chains to ensure two tips, since 0-value coinbase must be unspendable(and OP_RETURN by default, meaning the two chains would be otherwise identical).